io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java

```diff
@@ -1617,6 +1631,13 @@ private void writeProperties(Group parent, Collection<? extends ChildDefinition<
 							}
 						}
 
+						// special case handling: auto add codespace
+						if (getParameter(PARAM_ADD_CODESPACE).as(Boolean.class, true)) {
```

Comment:
If you check the parameter setting, if a codespace should be automatically added here, what is the function of the `addCodespace` parameter?

Reply:
?

Reply:
Maybe it's fine now?